### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by Setareh4IRAN** [WeAreMahsaAmini/Core#145] [Introductions]
Hey Everyone,
I'm here to help. I can search, collect data, translate and I'll be glad to do any other helpful tasks.
But I'm not familiar with this platform. How do you gather data? how should I share them with you?
thanks

**Comment by kaveh6** [comment]
here is the discord server , please join  : https://discord.gg/ZygYz7vQ

**Reply by Setareh4IRAN** [reply]
Thanks
